cmrt set CC=g++ -g in it make file. It is not right to use hard coded 'g++',
this patch fixes that.
--- work/cmrt-1.0.5/src/Makefile.am
+++ work/cmrt-1.0.5/src/Makefile.am
@@ -20,7 +20,7 @@
 # TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 # SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-CC = g++ -g
+CC = ${CXX} -g
 
 SUBDIRS =
 
